simulator_api/commands/communication-test.py

```diff
@@ -0,0 +1,50 @@
+"""Module that provides the Service command communication-test. The purpose of this module is to expose the capability of retrieving the status of communication with the Simulator container"""
+
+import requests
+from WebServerCore.ICommand import ICommand
+
+import simulator_api.utils.logger as logging
+from simulator_api.async_tasks.async_tasks import communication_test
+
+class CommunicationTest(ICommand):
+    """Service Command to retrieve the status of communication with simulator"""
+
+    def get_execute_latest(self, _url_params, task_id):
+        return self.get_execute_v1(_url_params, task_id)
+
+    def get_execute_v1(self, _url_params, task_id):
```

Comment:
add docstring and verify the _url_params validity

Reply:
_url_params as well as body data and url_specififcs are mandatory inputs when using WebServerCore